pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/pubsub/WorkflowStatusUpdate.java

```diff
@@ -77,10 +79,7 @@ public static void updateCustomWorkflow(Map<String, String> attributesMap, Strin
 
                 try {
                     if (isATRelatedStatusUpdate(studyGuid)) {
-                        boolean hasGenomicStudyGroup = participantDatas.stream().anyMatch(
-                                participantDataDto -> ATDefaultValues.GENOME_STUDY_FIELD_TYPE.equals(
-                                        participantDataDto.getFieldTypeId().get()));
-                        logger.info("ddpParticipantId: " + ddpParticipantId + " hasGenomicStudyGroup " + hasGenomicStudyGroup);
+                        log.info("uploading AT values for ddpParticipantId: ", ddpParticipantId);
```

Comment:
this might need to be logger instead of log?

Reply:
log is fine because of @Slf4j in line#30